strr-api/devops/vaults.gcp.env

```diff
@@ -5,9 +5,9 @@ AUTH_API_URL="op://API/$APP_ENV/auth-api/AUTH_API_URL"
 AUTH_API_VERSION="op://API/$APP_ENV/auth-api/AUTH_API_VERSION"
 LEGAL_API_URL="op://API/$APP_ENV/legal-api/LEGAL_API_URL"
 LEGAL_API_VERSION_2="op://API/$APP_ENV/legal-api/LEGAL_API_VERSION_2"
-LTSA_API_URL="op://API/$APP_ENV/ltsa/LTSA_API_URL"
-LTSA_API_VERSION="op://API/$APP_ENV/ltsa/LTSA_API_VERSION"
-LTSA_API_KEY_STRR="op://API/$APP_ENV/ltsa/LTSA_API_KEY_STRR"
+LTSA_API_URL="op://API/$APP_ENV/ltsa-api/LTSA_API_URL"
```

Comment:
do these 5 variables look right (and configured for dev/test environments)?

LTSA_API_URL="op://API/$APP_ENV/ltsa-api/LTSA_API_URL"
LTSA_API_VERSION="op://API/$APP_ENV/ltsa-api/LTSA_API_VERSION"
LTSA_API_KEY_STRR="op://API/$APP_ENV/ltsa-api/LTSA_API_KEY_STRR"
GEOCODER_API_URL="op://API/$APP_ENV/geocoder/GEOCODER_API_URL"
GEOCODER_API_AUTH_KEY="op://API/$APP_ENV/geocoder/GEOCODER_API_AUTH_KEY"

Reply:
they look good